.github/workflows/coverage.yml

```diff
@@ -1,9 +1,8 @@
 name: Test Coverage Uploader
 
 on:
-  push:
-    branches:
-      - master
+  schedule:
+    - cron:  '0 20 * * *'
```

Comment:
Lets stop running this workflow:
```
# added a workaround to stop running this
on:
  push:
    paths:
    - '**.never.ever.run'
```

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/10126/commits/f31a26dbfdd8436f0c2a6d20b334441d2772a183